code/datums/elements/poisonous.dm

```diff
@@ -0,0 +1,62 @@
+/**
+ * Poisonous component; which makes the attacks of the simplemob attached poison the enemy.
+ *
+ * Used for spiders and bees!
+ */
+/datum/component/poisonous
```

Comment:
In datums/elements? ✔
Is stateless? ✔
Actually a component? ✔

Yep, it's gaming time.

Reply:
Gotcha bitch